xeth/xeth.go

```diff
@@ -386,7 +386,7 @@ func (self *XEth) CurrentBlock() *types.Block {
 }
 
 func (self *XEth) GetBlockReceipts(bhash common.Hash) types.Receipts {
-	return self.backend.BlockProcessor().GetBlockReceipts(bhash)
+	return core.GetBlockReceipts(self.backend.ChainDb(), bhash)
```

Comment:
Ah, hmmm, thought they are also cached. Then it's ok like this. We can eventually benchmark this is we figure out a good way and then can add it if necessary.